include/TPP/Passes.td

```diff
@@ -270,4 +270,12 @@ def ElementWiseFusion : Pass<"element-wise-fusion", "func::FuncOp"> {
   let constructor = "mlir::tpp::createElementWiseFusionPass()";
 }
 
+def ConvInitSimplify : Pass<"conv-init-simplify", "func::FuncOp"> {
+  let summary = "Simplify initialization for convolution";
+  let description = [{
+    Rewrite the IR by doing some mathematical tricks.
```

Comment:
This really isn't a good description. The point here is to describe what to expect.

I'd say something like: "Simplifies convolution arguments and users by moving operations around to eliminate redundant computation and to promote instruction-level fusion at the TPP level".